tests/integration/questionnaires.test.js

```diff
@@ -257,6 +259,33 @@ describe('QUESTIONNAIRE ROUTES - GET /questionnaires/{_id}/follow-up', () => {
   let authToken;
   beforeEach(populateDB);
 
+  describe('TRAINING_ORGANISATION_MANAGER', () => {
```

Comment:
j'ai l'impression que ce test existe plus bas ligne 393 _should return 200 as user is ROF and route not called for a specific course_

Reply:
bien vu, j'ajoute les test que j'avais mis en plus du coup :)